keras/layers/convolutional_test.py

```diff
@@ -162,6 +162,12 @@ def fn(inpt):
       self.assertEqual(outp1_shape, layer(inpt1).shape)
 
 
+def test_conv1d_invalid_output_shapes(self):
+    kwargs = {'filters': 2, 'kernel_size': 20}
+    with self.assertRaises(ValueError):
```

Comment:
Any `assertRaises` should use  a regex to check the exception is actually the one you expect